test/integration_test.rb

```diff
@@ -0,0 +1,62 @@
+require 'helpers/kubeclient_helper'
+
+module KubernetesDeploy
+  class IntegrationTest < ::Minitest::Test
+    include KubeclientHelper
+
+    def setup
+      @logger_stream = StringIO.new
+      @logger = Logger.new(@logger_stream)
+      KubernetesDeploy.logger = @logger
+      @namespace = TestProvisioner.claim_namespace
+    end
+
+    def teardown
+      @logger_stream.close
+      TestProvisioner.delete_namespace(@namespace)
```

Comment:
It's very easy to reproduce:
```
༶ ktl get ns -w                               
NAME               STATUS    AGE
default            Active    135d
kube-system        Active    135d
NAME               STATUS    AGE
f0ea131043f23bfc   Active    6m
d934628332442544   Active    6m
```

That's two test runs with "Ctrl+C" in the middle.

https://github.com/seattlerb/minitest/issues/677
I'll give `Minitest.after_run { ... }` a try